src/io/flutter/module/FlutterSmallIDEGeneratorPeer.java

```diff
@@ -94,6 +100,10 @@ protected void textChanged(DocumentEvent e) {
     }
   }
 
+  public FlutterCreateAdditionalSettings getAddtionalSettings() {
+    return settingsFields.getAddtionalSettings();
```

Comment:
I'm guessing `buildUI` is sure to have been called before this is accessed and settingsFields is certain not to be null?